qa/workunits/rados/test_rados_tool.sh

```diff
@@ -534,12 +535,58 @@ function test_append()
   rm -rf ./rados_append_4k ./rados_append_4k_out ./rados_append_10m ./rados_append_10m_out
 }
 
+function test_put()
+{
+  # rados put test:
+  # replicated pool
+  $CEPH_TOOL osd pool create rados_put 100 100 replicated
```

Comment:
why not just use `$POOL`, which is already created in this test? if we do the cleanup right, there is no need to create another pool just for this test.